diff --git a/frida_tools/application.py b/frida_tools/application.py
index 3bd2597..0675815 100644
--- a/frida_tools/application.py
+++ b/frida_tools/application.py
@@ -688,7 +688,7 @@ class ConsoleApplication:
     def _pick_worker_pid(self) -> int:
         try:
             frontmost = self._device.get_frontmost_application()
-            if frontmost is not None and frontmost.identifier == "re.frida.Gadget":
+            if frontmost is not None and frontmost.identifier == "re.tr33newbee.Gadget":
                 return frontmost.pid
         except:
             pass
@@ -908,7 +908,7 @@ class ConsoleApplication:
         return result[0]
 
     def _get_default_frida_dir(self) -> str:
-        return os.path.join(os.path.expanduser("~"), ".frida")
+        return os.path.join(os.path.expanduser("~"), ".tr33newbee")
 
     def _get_windows_frida_dir(self) -> str:
         appdata = os.environ["LOCALAPPDATA"]
diff --git a/frida_tools/repl.py b/frida_tools/repl.py
index f2a516e..08f9787 100644
--- a/frida_tools/repl.py
+++ b/frida_tools/repl.py
@@ -943,7 +943,7 @@ URL: {url}
                 os.rename(old_file, new_path)
                 return True
 
-        old_file = os.path.join(os.path.expanduser("~"), ".frida", name)
+        old_file = os.path.join(os.path.expanduser("~"), ".tr33newbee", name)
         if os.path.isfile(old_file):
             os.rename(old_file, new_path)
             return True
diff --git a/scripts/meson.build b/scripts/meson.build
index cec36d6..bcad3bf 100644
--- a/scripts/meson.build
+++ b/scripts/meson.build
@@ -1,5 +1,5 @@
 scripts = [
-  ['frida', 'repl'],
+  ['tr33newbee', 'repl'],
   ['frida-apk', 'apk'],
   ['frida-compile', 'compiler'],
   ['frida-create', 'create'],
